cachi2/core/package_managers/pip.py

```diff
@@ -1467,7 +1485,7 @@ def _download_dependencies(
                     must_match_any_checksum(pkg.path, pkg.checksums_to_verify)
             except PackageRejected:
                 pkg.path.unlink()
-                log.warning("The %s is removed from the output directory", pkg.path.name)
+                log.warning("Download '%s' was removed from the output directory", pkg.path.name)
```

Comment:
This line change needs to be reflected in the unit tests otherwise they fail.

Reply:
It is? https://github.com/ben-alkov/cachi2/blob/df11af3d29e70f4d1dd3c002cfd8f93e4c29cb78/cachi2/core/package_managers/pip.py#L1475

Reply:
Wrong commit though: https://github.com/containerbuildsystem/cachi2/commit/df11af3d29e70f4d1dd3c002cfd8f93e4c29cb78#diff-4c1241d88628d8032e8836ab6fa96853ce996ad356da838eca9a58daece89c4aR3557

You need the change in this commit too.

Reply:
Either way, it seems to be in the correct state now.